lib/static/css/main.css

```diff
@@ -1830,13 +1830,17 @@ footer .social {
   padding: 5px 0;
 }
 
-.entry-share {
+.blog-social {
   padding: 36px 0;
   display: block;
 }
 
-@media screen and (max-width: 735px) {
-  .entry-share {
+.blog-social div {
```

Comment:
It's not a good practice to use raw tags in CSS as you may accidentally target unintended elements (be it existing or in the future). Add a class to those the specific `div`s you want to target.

Reply:
ok 👍